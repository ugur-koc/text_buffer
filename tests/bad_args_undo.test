undo bad_arg
exit