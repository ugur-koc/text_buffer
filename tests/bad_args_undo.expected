Available Operations: insert append eraseAt erase replace load save help exit 
Text Buffer: 
Arguments to undo are ignored!

Nothing to undo

Text Buffer: 
